### PR TITLE
Adds methods for getting validator errors.

### PR DESCRIPTION
**Pull request by pietgeursen**
Depject makes this kinda hard and this seems like an ok solution just because it's really only for debugging.

- Not sure if the naming is what I want. I think `parseErrors` is better than `isPollError` though.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -7,7 +7,17 @@ var isNormalisedPoll = Validate(pollSchema)
...
+  isPoll,
+  parsePoll,
+  parsePollErrors,
+
+  isPosition,
+  parsePosition,
+  parsePositionErrors,
+
+  versionStrings
+} = require('../')
```
Comment (mixmix) [inline test/index.test.js]: this looks like a christmas tree :christmas_tree: 

#5amCodeReviews!

File: v1/index.js [hunk 2]
```diff
@@ -29,11 +31,13 @@ module.exports = {
     return nest({
       poll: {
         parse: parsePoll,
+        parseErrors: parsePollErrors,
```
Comment (mixmix) [inline v1/index.js]: this doesn't need nest!

File: README.md [hunk 3]
```diff
@@ -51,6 +51,11 @@ console.log(isPoll(validPoll)) // => true
 - `parsePoll`,
 - `parsePosition`,
 
+### Parser Errors (useful for debugging)
+
+- `parsePollErrors`,
+- `parsePositionErrors`,
```
Comment (mixmix) [inline README.md]: can you give me some sample return values e.g. 

```js
parsePollErrors(poll)
=> {
  errors: {
    v1:  // .. missing a title!
    v2:  // .. missing new attribute added in v2! 
  }
}
```

Comment (mixmix) [reply README.md]: I think calling it `getPollErrors` might be more descriptive and less confusing. feels active like yeah I am really just wanting to get all those errors plz

File: v1/index.js (hunk 2 above)
Comment (pietgeursen) [reply v1/index.js]: Why not? What's the alternative?

File: README.md (hunk 3 above)
Comment (pietgeursen) [reply README.md]: Nice, just changed the naming to `getPollErrors` as you suggest. Way better.

**Pull request merged**